<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmGestionCobros2</class>
 <widget class="QDialog" name="FrmGestionCobros2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Desglose de cobro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 253, 215);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 127);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Importe Pendiente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 104, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Transferencia</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="2">
    <widget class="QLineEdit" name="txtPendiente">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 104, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Cheque</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="txtCheque">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <widget class="QLineEdit" name="txtCambio">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QPushButton" name="btnAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
     <property name="icon">
      <iconset resource="../maya.qrc">
       <normaloff>:/Icons/PNG/OK.png</normaloff>:/Icons/PNG/OK.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="txtImporte_pendiente">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="txtEfectivo">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 104, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tarjeta cr√©dito</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../maya.qrc">:/Icons/PNG/pagos.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 104, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Internet (PyPal - Otros)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="txtTransferencia">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 127);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Entrega</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Cambio</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 104, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Efectivo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../maya.qrc">
       <normaloff>:/Icons/PNG/Delete.png</normaloff>:/Icons/PNG/Delete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="txtTarjeta">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="txtInternet">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Pendiente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QLineEdit" name="txtEntrega">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtEfectivo</tabstop>
  <tabstop>txtTransferencia</tabstop>
  <tabstop>txtTarjeta</tabstop>
  <tabstop>txtCheque</tabstop>
  <tabstop>txtInternet</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections/>
</ui>
